DataFormats/L1TGlobal/interface/GlobalObject.h

```diff
@@ -24,6 +24,7 @@ enum GlobalObject
     gtHTT,
     gtHTM,
     gtETMHF,
+    gtTowerCount,
```

Comment:
Hmm, I am not sure.  
Examples from the past 6be8e4d7e7b1d0ed4000662119ac002ea5239961, 3f17f45de068a6a5bb38f9bf1ea2c3f2ad606ce2.